index.js

```diff
@@ -20,6 +20,16 @@ var mainQueue = new TaskQueue();
 var tempViewQueue = new TaskQueue();
 var CHANGES_BATCH_SIZE = 50;
 
+var coffee;
+try {
+  coffee = require('coffee-script');
```

Comment:
This won't work with browserify you need to add coffee script:false to the package.json